### PR TITLE
Update the text for delete key press in DeleteKeyModal

### PR DESCRIPTION
**Pull request by akanshgulati**
- Update the text shown in alert on Delete key step, fixes issue #320 
From `We are now deleting the key for your address.`
To: `The key for your address is now being deleted.`

### CODE REVIEW

File: containers/keys/deleteKey/DeleteKeyModal.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ const DeleteKeyModal = ({ onClose, Address, addressKeys, KeyID, privateKey, ...r
             return {
                 submit: c('Action').t`Done`,
                 loading: true,
-                children: <Alert>{c('alert').t`We are now deleting the key for your address.`}</Alert>
+                children: <Alert>{c('alert').t`The key for your address is now being deleted..`}</Alert>
```
Comment (mmso) [inline containers/keys/deleteKey/DeleteKeyModal.js]:
```suggestion
                children: <Alert>{c('alert').t`The key for your address is now being deleted.`}</Alert>
```

**Comment by mmso** [comment]
This was not in the original issue, but could you please also update the text `We are now generating encryption keys...` in the AddKeyModal.js to be consistent with this new format?

**Comment by akanshgulati** [comment]
@mmso 
From: `We are now generating encryption keys for your address, this may take several minutes and temporarily freeze your browser.`
To: `The encryption keys for your address are being generated. This may take several minutes and temporarily freeze your browser.`

Please confirm.

**Comment by mmso** [comment]
Sounds good to me @akanshgulati

**Comment by akanshgulati** [comment]
@mmso  Done.

**Comment by mmso** [comment]
Thanks for you help @akanshgulati !

**Comment by akanshgulati** [comment]
@mmso  any playground to try these components or Proton mail repository which is using it to try different steps? Help needed.

**Comment by mmso** [comment]
For this specific issue, you can clone and run the https://github.com/ProtonMail/proton-mail-settings project to see the result. If you want to see changes locally, you can use `npm link` with for example react-components.

**Pull request merged**